### DISCUSSION TITLE
FCP extension no longer shows up

### DISCUSSION THREAD

**Discussion by timcorbin** [latenitefilms/BRAWToolbox#47] [Support] [answered]
I installed from Test Flight a couple weeks ago and everything worked fine, was able to import BRAW files just fine. I came back to my machine today and the extension is now gone. Uninstall and reinstall changes nothing. Any ideas?

**Comment by latenitefilms** [comment]
Is it still installed in TestFlight? Maybe there's an update you need to install?

Also see:

https://brawtoolbox.io/troubleshooting/#braw-toolbox-workflow-extension-doesnt-appear-in-final-cut-pro

**Reply by timcorbin** [reply, marked as answer]
So it's working again after removing and reinstalling FCP. It's strange in that it hid all extensions from me except Frame.io. Once I reinstalled I have all my other extensions back too.

**Reply by timcorbin** [reply]
And while it is "working", half the imports beach ball FCP, and when they do import, playback on the imported clip is really choppy and it will frequently throw up the red 'Failed to Process BRAW File' error in the viewer. MacOS 12.6.2 FCP 10.6.5.

**Reply by latenitefilms** [reply]
That's strange.

What if you lower the decode quality? Where you just using rhe default "Automatic (HD)"?

What are your system specs? It's not a Trashcan Mac Pro by any chance?

How fast is the hard drive containing the BRAW files?

Does the same files play correctly in the Workflow Extension? If not, does it play correctly in Blackmagic RAW Player or DaVinci Resolve?
